Hestia/app/src/main/java/hestia/UI/dialogs/IpDialog.java

```diff
@@ -49,4 +51,9 @@ void pressConfirm() {
                     Toast.LENGTH_SHORT).show();
         }
     }
+
```

Comment:
Check my ip branch pull request for How to use Java Patterns (regex) to check this. If I have the time today i Will add it.